lib/modules/manager/bundler/extract.ts

```diff
@@ -55,9 +55,7 @@ export async function extractPackageFile(
       };
       if (gemMatch.groups?.currentValue) {
         const currentValue = gemMatch.groups.currentValue;
-        dep.currentValue = regEx(/\s*,\s*/).test(currentValue)
-          ? currentValue
-          : currentValue.slice(1, -1);
+        dep.currentValue = currentValue;
```

Comment:
i think we should trim the quotes here instead of handling them in versioning.

quotes should never been part of current value.

Reply:
> quotes should never been part of current value.

Sure, I'll try to figure out how to handle the current value in that way. Note that we do not currently remove quotes if there are multiple constraints.